app/uk/gov/hmrc/catalogue/github/GithubV3ApiClient.scala

```diff
@@ -31,6 +31,8 @@ import scala.util.{Failure, Success, Try}
 trait GithubV3ApiClient {
   self : GithubEndpoints with GithubCredentialsProvider  =>
 
+  def isInternal : Boolean
```

Comment:
agreed with you and was really not happy to put it here. but the previous way of looking at the url to figure out wether its internal repo or not was also not great. I can look at it later again.